srcDoc/command_line_parameters.rst

```diff
@@ -345,61 +345,61 @@ Test Parameters for Publishing and Subscribing Applications
 
 -  ``-peer <address>|<addres>[:<id>]``
 
-   Adds a peer to the peer host address list. If ``-rawTransport`` is used, a
-   optional ID of the subscriber could beprovied. This argument may be repeated
+   Adds a peer to the peer host address list. If ``-rawTransport`` is used, 
```

Comment:
Second comment from the Perftest.3.0.0 edits. This argument means that "-peer <a>" can be repeated:

-peer a -peer b -peer c

Reply:
I just noticed that the second "addres" above is missing an 's' (<address>|<addres>)

Reply:
Then I would recommend saying:
This argument may be repeated to indicate multiple peers. For example: -peer a -peer b -peer c.

(Should we actually use an address in the example to make it clearer?)

Reply:
Good point, yes, I think we should go with something like what you wrote:

```This argument may be repeated to indicate multiple peers. For example: -peer 1.1.1.1 -peer 2.2.2.2 -peer 3.3.3.3.
```

Reply:
done